app/services/dttp/retrieve_account.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Dttp
+  class RetrieveAccount
+    include ServicePattern
+
+    class HttpError < StandardError; end
+
+    attr_reader :provider
+
+    ACCOUNT_ATTRIBUTE_FILTERS = %w[
+      name
+      accountid
+      dfe_ukprn
```

Comment:
Just checking - are we pulling out `dfe_ukprn` and `dfe_awardinginstitution` for a specific reason?

Reply:
No particular reason, the Trello ticket stated: A basic set or properties for starters is fine in this ticket. I guess we can change it depending on what we need later?

Reply:
Ok! Let's remove them for now and just keep name, since this is all our database knows about right now. Otherwise, I'll bet they'll stay in forever 😂

Reply:
Fair point. Done. 😂